babel.config.js

```diff
@@ -356,6 +366,11 @@ function importInteropTest(source) {
   if (source === "@babel/helper-plugin-test-runner") {
     return "none";
   }
+  // This file is native ESM, but we compile it to CJS in old versions of Node.
+  // TODO: Remove this case once we only test ESM.
+  if (source === "strip-ansi") {
+    return "babel";
+  }
```

Comment:
IMO compiling node_modules is dangerous as we will lose the guard that we, or any of our transitive dependencies are using `strip-ansi` version that is not supported by our minimum node versions. I know that currently `strip-ansi` is only used in `devDependencies` so here is the future tense.

Currently `strip-ansi` is used for processing the coloured output from babel/code-frame. The `strip-ansi` is designed to strip most terminal escapes, including colour escapes. For our case, perhaps we can switch to https://www.npmjs.com/package/stripcolorcodes? This library only work for colour escapes so the regex is simpler than the one in `ansi-regex`.

Reply:
`strip-ansi` would still be in our dependencies tree anyway -- I'll just close this PR 👍